diff --git a/knot_exporter/knot_exporter.py b/knot_exporter/knot_exporter.py
--- a/knot_exporter/knot_exporter.py	(revision af7b7c54ffee66fa12b2c8da4eb39d5251199866)
+++ b/knot_exporter/knot_exporter.py	(date 1708894640123)
@@ -12,6 +12,7 @@
 import libknot.control
 
 from prometheus_client.core import REGISTRY
+from prometheus_client.core import CounterMetricFamily
 from prometheus_client.core import GaugeMetricFamily
 from prometheus_client.exposition import MetricsHandler
 
@@ -71,8 +72,11 @@
 
         def metric_families_append(family, labels, labels_val, data):
             m = metric_families.get(family, GaugeMetricFamily(family, '', labels=labels))
+            c = metric_families.get(family + '_total', CounterMetricFamily(family, '', labels=labels))
             m.add_metric(labels_val, data)
+            c.add_metric(labels_val, data)
             metric_families[family] = m
+            metric_families[family + '_total'] = c
 
         if self.collect_meminfo:
             # Get global metrics.
